src/components/InventoryManager/Sidebar/InventoryManagerSidebar.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Grid, Typography, Divider } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import FilterMenu from './FilterMenu';
+import PriceMenu from './PriceMenu';
+import SortMenu from './SortMenu';
+
+const useStyles = makeStyles({
+  marketplaceText: {
+    fontFamily: 'Work Sans',
+    fontSize: '1rem',
+    color: '#373737',
+  },
+  filterText: {
+    fontFamily: 'Work Sans',
+    fontSize: '1.5rem',
+    color: '#373737',
+  },
+  pageText: {
+    fontFamily: 'Work Sans',
+    fontSize: '1rem',
+    fontWeight: 'bold',
+  },
+  divider: {
+    width: '80%',
+  },
```

Comment:
Can you make the all of the sidebar fonts a little smaller + more relative to the size in the figma?

Reply:
ooh sorry about this but for the sidebar, most of the changes are in the InventoryManagerAirlock and FilterSidebar pull request, I just moved on to this feature without going back to the master branch so it carried over the changes, which is why all those files are in here unfortunately. But for the styling of the sidebar, I've made some changes in the other PR!

Reply:
Ok, got it!